api-docs/rst/dev-guide/concepts.rst

```diff
@@ -798,3 +808,36 @@ To get started with RackConnect v3 cloud bursting:
 
 .. _RackConnect product information: http://www.rackspace.com/cloud/hybrid/rackconnect
 .. _Knowledge Center articles for RackConnect: http://www.rackspace.com/knowledge_center/product-page/rackconnect
+
+
+.. _concergence-concept:
+
+Convergence
+~~~~~~~~~~~
+
+Convergence is completely rewritten orchestration layer of autoscale that provides
+higher reliability for scaling up and down by optimizing use of Nova API with retries
+as necessary. In addition, Convergence continuously keeps track of all the servers
+in autoscaling groups, notices when a server is deleted out-of-band or goes into
+an non-ACTIVE state (eg ERROR) and automatically replaces it.
+We call this capability “self-healing”.
+
+A new endpoint :ref:`**converge** <post-create-scaling-group-v1.1-tenantid-converge>`
+is added which triggers convergence on a group. This can be useful when a group's
+status is ERROR which happens when group has invalid launch configuration
+(ex imageRef of deleted image). After correcting the launch configuration, one
+needs to invoke this endpoint to converge the group.
+
+With this change there are minor behavioral differences which should not impact
+any existing users:
+
+- If autoscaled server is manually removed from load balancer it is supposed
+  to be on as per scaling group config, then autoscale will revert that change
+  and put the server back in configured CLB. Note that autoscale does not care
+  if server is added to any other CLB. It only ensures that server is always
+  there in configured CLB.
+
+- If a CLB configured in the group is not found / deleted then the group will
+  be put in ERROR and server that was supposed to be added to CLB *will* remain.
+  This is unlike current behavior where the server that couldn't be added to CLB
+  gets deleted.
```

Comment:
The information about the behavioral difference belongs in the Release Notes.   

If you want people reading about the Convergence concept to be aware of this information, you can add a note linking to the "behavioral changes" section of the release notes.